docs/mathematics/armstrong-number.mdx

```diff
@@ -0,0 +1,172 @@
+---
+title: Armstrong Number
+slug: /mathematics/armstrong-number
+---
+
+## Introduction
+
+An Armstrong Number is an n-digit number whose sum of the nth powers of the number is equal to the number itself. Other names include Narcissistic Number, Perfect Digital Invariant, and Plus Perfect Number.
+
+## Explanation
+
+Let `pqrs` be a number. Then, 
+
+<a href="https://www.codecogs.com/eqnedit.php?latex=(&space;p^4&space;&plus;&space;q^4&space;&plus;&space;r^4&space;&plus;&space;s^4&space;)&space;=&space;pqrs" target="_blank"><img src="https://latex.codecogs.com/gif.latex?(&space;p^4&space;&plus;&space;q^4&space;&plus;&space;r^4&space;&plus;&space;s^4&space;)&space;=&space;pqrs" title="( p^4 + q^4 + r^4 + s^4 ) = pqrs" /></a>
+
+Since, `pqrs` is a 4-digit number, n = 4. This is why each digit is raised to the power of 4.
+#### Quick Example : 
+Let 1634 be the number taken. Here 1634 is a four-digit number, therefore n = 4.
+
+<a href="https://www.codecogs.com/eqnedit.php?latex=(1^4&space;&plus;&space;6^4&space;&plus;&space;3^4&space;&plus;&space;4^4)&space;=&space;(1&plus;1296&plus;81&plus;256)&space;=&space;1634" target="_blank"><img src="https://latex.codecogs.com/gif.latex?(1^4&space;&plus;&space;6^4&space;&plus;&space;3^4&space;&plus;&space;4^4)&space;=&space;(1&plus;1296&plus;81&plus;256)&space;=&space;1634" title="(1^4 + 6^4 + 3^4 + 4^4) = (1+1296+81+256) = 1634" /></a>
+
+Result : 1634 is an Armstrong Number.
+
+## Algorithm
+
+ - Start
+ - Input number `num`.
+ - Calculate the number of digits in `num`. Let this be stored in `order`.
+ - For each digit `d` in `num`, compute `d` raised to the power of `order`.
+ - Calculate the sum of each `d` to the power of `order` for all the digits in `num` and store in variable `sum`.
+ - Compare `sum` and `num`. If equal, return `An Armstrong Number`. Else, return `Not an Armstrong Number`.
+ - End.
```

Comment:
No need to have this as well

Reply:
> No need to have this as well

Do i need to remove all of the above lines and enter the algorithm in a single step or remove only the end step.

Reply:
Remove only the end step.